### ISSUE TITLE
Block Submenu Layout

### ISSUE THREAD

**Issue by andreyasko** [unacms/UNA#518]
Block submenu is currently always hidden behind chevron without any clues on available sorting/filtering options or actions.

Whenever possible, show block submenu as links/tabs, and when space is limited convert into our standard pop-down menu with chevron, but use currently chosen item as a visible label. Use chevron-only display if there's no space for a current item label.

![image](https://cloud.githubusercontent.com/assets/22210428/20953266/43be352a-bc86-11e6-8f6b-7581588cc37a.png)

**Comment by AlexTr** [comment]
Such remake will require some significant changes in the code since it supposed to be done not in comments only, but site wide.

As I remember these options in the comments were supposed to be hidden by default without ability to change it by user. We still leave it in comments, but I think more visibility isn't needed here. 

Also such menu is very common nowdays, for example on facebook on desktop version every block has such chevron styled menu.

**Comment by andreyasko** [comment]
Yes, chevron menu is common, but it is used for actions that can be done to current block. It's not quite suitable to tabs-like menu that is supposed to switch content of the block (like we have in Manage blocks for admins) or for filtering menus, unless it is annotated. User needs to see browsing options to know that they exist within this block. 

For example, when simply chevron is used in comments block I wouldn't know if there're any fildering options. I'd thing it is to manage the block (report, hide, etc. ). 

In general I suggest that when there is enough available space we show items as full links, if not enough - as chevron with current selection label. If chevron menu only has actions (no current state) - use pure chevron.

**Comment by AlexTr** [comment]
Most probably it will require significant remake in many places.

I will check if it can be done universally in some system files, if not - we need to postpone it for later version and reconsider this functionality.
